sound/soc/intel/common/soc-acpi-intel-adl-match.c

```diff
@@ -295,6 +333,13 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_adl_sdw_machines[] = {
 		.drv_name = "sof_sdw",
 		.sof_tplg_filename = "sof-adl-rt711.tplg",
 	},
+	{
+                .link_mask = 0x5, /* rt5682 on link0 & 2xmax98373 on link 2 */
+                .links = adl_chromebook_base,
+                .drv_name = "sof_sdw",
+                .sof_fw_filename = "sof-adl.ri",
+                .sof_tplg_filename = "sof-adl-sdw-max98373-rt5682.tplg",
```

Comment:
this will require a topology permutation if indeed this is the case that link2 is used. Do you have a PR for the topology change?

Reply:
Sure , i will create a PR with topology changes.

Reply:
Created the PR for Topology changes
https://github.com/thesofproject/sof/pull/4165